vagrant/guest/etc/nginx/conf.d/sites.conf

```diff
@@ -1,15 +1,5 @@
 #
-# The default server
+# Include site configuration files
 #
-server {
```

Comment:
Removing this breaks custom nginx configs which only implement a secure server config (thus relying on this for passing through to the server backend)